lib/patches/thread_safe/thread_safe.rb

```diff
@@ -0,0 +1,180 @@
+Truffle::Patching.require_original __FILE__
```

Comment:
Maybe we should add a github link to a specific fixed revision in patch files, to make it easy to see the original file, and to track which commit this was taken from.